activity_calendar/models.py

```diff
@@ -56,6 +56,7 @@ class Meta:
     description = MarkdownTextField(help_text="Note that uploaded images are publicly accessible, even if the activity is unpublished.")
     location = models.CharField(max_length=255)
     image = models.ForeignKey(PresetImage, blank=True, null=True, related_name="activity_image", on_delete=models.SET_NULL)
```

Comment:
You could rename `image` to `slots_image`. Either way, adding in some help_text would be nice